server/scripts/import-tests/index.js

```diff
@@ -53,10 +53,18 @@ const builtTestsDirectory = path.resolve(gitCloneDirectory, 'build', 'tests');
 const testsDirectory = path.resolve(gitCloneDirectory, 'tests');
 
 const gitRun = (args, cwd = gitCloneDirectory) => {
-    return spawn
-        .sync('git', args.split(' '), { cwd })
-        .stdout.toString()
-        .trimEnd();
+    const gitRunOutput = spawn.sync('git', args.split(' '), { cwd });
+
+    if (gitRunOutput.error) {
+        console.info(
+            `'git ${args.split(' ')} ${cwd}' failed with error ${
```

Comment:
Splitting on the empty space is necessary for `spawn.sync`. Here, though, by coercing the resultant array back into a string, we'll end up with a misrepresentation of the command executed (e.g. `git fetch origin main` becomes `git fetch,origin,main`).

Reply:
Thanks for catching this! I've updated the code to remove the split.